airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/requester.py

```diff
@@ -117,7 +117,7 @@ def get_request_body_json(
         stream_state: Optional[StreamState] = None,
         stream_slice: Optional[StreamSlice] = None,
         next_page_token: Optional[Mapping[str, Any]] = None,
-    ) -> Optional[Mapping[str, Any]]:
+    ) -> Mapping[str, Any]:
```

Comment:
this method signature is not conform to its parent definition https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/request_options/request_options_provider.py#L68-L74